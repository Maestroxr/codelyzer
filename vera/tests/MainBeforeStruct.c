
/*OK*/
struct first
{
	int something;
};
/*********/



int main() 
{
	return 0;
	
}


/*PROBLEM*/
struct second
{
	int somethingElse;
};
/*********/
