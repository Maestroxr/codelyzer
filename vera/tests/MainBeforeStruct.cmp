C:\Work\AutoAssessment\codelyzer\vera\tests\MainBeforeStruct.c:19: Main function appears before struct defintion. struct is at line 19 , main appeared at line 11.