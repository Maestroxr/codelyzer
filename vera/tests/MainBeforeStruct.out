C:\Work\AutoAssessment\codelyzer\vera\tests\MainBeforeStruct.c:3: Curly bracket tab indentation incorrect, found 12 characters but expected 0 tabs.
C:\Work\AutoAssessment\codelyzer\vera\tests\MainBeforeStruct.c:5: Closing curly bracket not in the same line or column, or brackets are not both indented with tabs.
C:\Work\AutoAssessment\codelyzer\vera\tests\MainBeforeStruct.c:18: Main function appears before struct defintion. struct is at line 18 , main appeared at line 10.
C:\Work\AutoAssessment\codelyzer\vera\tests\MainBeforeStruct.c:18: Curly bracket tab indentation incorrect, found 13 characters but expected 0 tabs.
C:\Work\AutoAssessment\codelyzer\vera\tests\MainBeforeStruct.c:20: Closing curly bracket not in the same line or column, or brackets are not both indented with tabs.
